package.json

```diff
@@ -48,6 +50,7 @@
     "jsdom-testing-mocks": "^1.13.0",
     "msw": "^2.3.4",
     "msw-storybook-addon": "^2.0.3",
+    "playwright": "^1.46.0",
```

Comment:
Is there a way to configure playwright so that `pnpm exec playwright install` installs only chromium? A config file for playwright for example.

Reply:
AFAIK there is no way to configure it with a config file, you need to run `pnpm exec playwright install chromium`.

However, instead of `playwright`, we could install `playwright-chromium` package. This downloads chromium automatically. The only downside is that `pnpm install` will take longer.

Reply:
Hmm that seems excessive, seeing how 95% of people executing `pnpm i` in this project are never going to use playwright.

What I really wanted to say is that we should be fine with running those tests only in Chromium – I assume this is going to make them faster. It looks like `test-storybook` has a CLI flag for this.

Reply:
They are only run in Chromium by default, anyway, but I added an explicit flag just in case if it's just my environment.